deps/readline.lua

```diff
@@ -396,7 +396,7 @@ local keyHandlers =
     self:getHistory(10)
   end},
   -- Printable characters
-  {{function(key, char) return char > 31 and key:sub(1,1) or nil end}, function(self, consumedKeys)
+  {{function(key, char) return char > 31 and key or nil end}, function(self, consumedKeys)
```

Comment:
`key:sub(1,1)`suppose that the printable character is 1 byte length,but unfortunately,Lot's of unicode character isn't 1-byte-wide.This sub only gets the first byte of the character.

Maybe we need a function `chlen()` to get the length of a unicode character and the code will be like `key:sub(1,ustring.chlen(key))`